lib/subscription/subscription-store.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference path='../../typings/tsd.d.ts' />
+
+import _ = require('lodash');
+import Subscription = require('./subscription');
+
+export = SubscriptionStore;
+
+class SubscriptionStore<T extends Subscription> {
```

Comment:
I'll defer on this until the polling changes come in.  For the rest of the file, let's clean up she spacing around the parameters in the declarations and the the type declarations (i.e., remove the space padding).  I will have PR soon so that `tslint` captures these things.